pkg/server/endpoints/node/handler_test.go

```diff
@@ -60,7 +60,7 @@ type HandlerTestSuite struct {
 	now              time.Time
 }
 
-func SetupHandlerTest(t *testing.T) *HandlerTestSuite {
+func SetupHandlerTest(t *testing.T, attestorBaseName, resolverBaseName string) *HandlerTestSuite {
```

Comment:
rather than requiring each test to pass these values, i think it might be cleaner and clearer if `SetupHandlerTest` adds the node attestor using some default name, like "fake" or something, and then all of the other tests don't bother adding a resolver. Then there can be a test that adds a resolver under the right name, and one that adds a resolver under the wrong name (and depending on the rest of the tests, a test that has no resolver).

All of the above assumes you add a new field to the suite to hold onto the catalog...

Reply:
Yes, I totally agree!